Formula/gnutls.rb

```diff
@@ -20,6 +21,7 @@ class Gnutls < Formula
   depends_on "nettle"
   depends_on "guile" => :optional
   depends_on "unbound" => :optional
+  depends_on "p11-kit"
```

Comment:
Given we've only _just_ restored this from the boneyard, we might want to make this `:optional` for now.